### PR TITLE
[Scaling] Use more tolerant checks for testing

### PR DESCRIPTION
**Pull request by himani2411**
### Description of changes
* Use tolerant check as some nodes while scaling are throttling API's 
* Do not retry if scaling target has been achieved
* Reduce MaxQueue Size to 5K

### Tests
Static Scaling Stress test

### References
* Link to impacted open issues.
* Link to related PRs in other packages (i.e. cookbook, node).
* Link to documentation useful to understand the changes.

### Checklist
- Make sure you are pointing to **the right branch**.
- If you're creating a patch for a branch other than `develop` add the branch name as prefix in the PR title (e\.g\. `[release-3.6]`).
- Check all commits' messages are clear, describing what and why vs how.
- Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hanwen-pcluste — APPROVED** [review]

File: tests/integration-tests/tests/common/scaling_common.py [hunk 1]
```diff
@@ -58,8 +58,8 @@ def retry_if_scaling_target_not_reached(
     # Return True if we should retry, which is when the target cluster size
     # (either EC2 or scheduler compute nodes) is not reached yet
     return (
-        (use_ec2_limit and ec2_capacity_time_series[-1] != target_cluster_size)
-        or (use_compute_nodes_limit and compute_nodes_time_series[-1] != target_cluster_size)
+        (use_ec2_limit and max(ec2_capacity_time_series) != target_cluster_size)
+        or (use_compute_nodes_limit and max(compute_nodes_time_series) != target_cluster_size)
```
Comment (hanwen-cluster) [inline tests/integration-tests/tests/common/scaling_common.py:62]: Why is this necessary?

Comment (himani2411) [reply tests/integration-tests/tests/common/scaling_common.py:62]: The *_time_series is a list where we were previously checking just the last entry.
 However, from the logs I could 
 see that we did reach the target and later lost one of the nodes due to some ec2 scheduling errors or other errors.

Comment (hanwen-cluster) [reply tests/integration-tests/tests/common/scaling_common.py:62]: In your described case, the scaling failed. So we should treat it as failure, right?
